PokemonGo.RocketAPI/Rpc/Encounter.cs

```diff
@@ -56,7 +56,7 @@ public class Encounter : BaseRpc
             return await PostProtoPayload<Request, CatchPokemonResponse>(RequestType.CatchPokemon, message);
         }
 
-        public async Task<IncenseEncounterResponse> EncounterIncensePokemon(long encounterId, string encounterLocation)
```

Comment:
I did a modification at POGOprotos so another users will not be able to compile with ulong. ( I also did a pull request ). Please, reverse this commit, and sorry. :)